internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -3205,10 +3188,7 @@ func resourceKubernetesClusterDelete(d *pluginsdk.ResourceData, meta interface{}
 		}
 	}
 
-	ignorePodDisruptionBudget := true
-	err = client.DeleteThenPoll(ctx, *id, managedclusters.DeleteOperationOptions{
-		IgnorePodDisruptionBudget: &ignorePodDisruptionBudget,
-	})
```

Comment:
@ms-henglu could you find out from the service team what the default behaviour for delete operations on the stable API version is? Are PDBs respected by default or ignored?

Reply:
refs: https://learn.microsoft.com/en-us/azure/aks/delete-node-pool?tabs=azure-cli

Reply:
Thank you for the confirmation 👍